src/iquest.cpp

```diff
@@ -279,8 +279,8 @@ execAndShowSpecificQuery( rcComm_t *conn, char *sql,
     }
     status = rcSpecificQuery( conn, &specificQueryInp, &genQueryOut );
     if ( status == CAT_NO_ROWS_FOUND ) {
-        printf( "No rows found\n" );
-        return 0;
+        printf( "CAT_NO_ROWS_FOUND: Nothing was found matching your query\n" );
```

Comment:
is this message printed twice?

i noticed it is also printed to the terminal on line 404.

Reply:
oh, i see - this inner one (execAndShowSpecificQuery ) will already have printed by the time 404 gets hit... so i can just remove the 404 printf() and keep the return values correct.

Reply:
seems reasonable

Reply:
done

Reply:
is there any particular reason to prefer printing the message on line 282 rather than on line 404?

Reply:
i don't think so - just that it happens 'closer' to the query.

shouldn't matter where - can move if you like it 'outside'.

Reply:
no. i was just wondering.